root/programs/C#/Frameworks/Infrastructure/Business/RichClient/Business/MyBaseLogic2CS.cs

```diff
@@ -157,7 +157,7 @@ public abstract class MyBaseLogic2CS : BaseLogic2CS
                 else if (parameterValue.ActionType.Split('%')[0] == "ODP")
                 {
                     // Oracle / ODP.NET用のDamを生成
-                    dam = new DamOraOdp();
+                    dam = new DamManagedOdp();
 
                     // 接続文字列をロード（ODP2：Instant Client）
                     connstring = GetConfigParameter.GetConnectionString("ConnectionString_ODP2");
```

Comment:
@daisukenishino 

What is the meaning of ODP*2*?
You might as well rename *ODP2* to *ODP*.

Reply:
@daisukenishino 

In [testScreen.aspx](https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/VB/Samples/WebApp_sample/WebForms_Sample/WebForms_Sample/Aspx/testPublic/testScreen.aspx#L154), the words called *ODP1* or *ODP2* are still remain.